### DISCUSSION TITLE
useVirtualList for a grid layout with 3 items in a row

### DISCUSSION THREAD

**Discussion by ghost** [vueuse/vueuse#2777] [Q&A]
Is it possible to use `useVirtualList` for a list, that has 3 items in a row?

Like
```
1 2 3
4 5 6
7 8 9
…
```

instead of

```
1
2
3
4
…
```

I am not able to wrap the rows in `<div>`’s, because I need to change the column count for mobile devices — smaller screens will have 2 items in a row and tiny screens 1 item a row. Done with css grid.

I think, this is a common use case.

Does it work with `useVirtualList`?

**Comment by Mateleo** [comment]
I would like to know too

**Comment by JeromeD3** [comment]
so， what is the solution

**Comment by z0ffy** [comment]
Hello, I have a warehouse that simply implements the function you mentioned, you can refer to it briefly.

https://github.com/667-1/virtual-list
